tests/conftest.py

```diff
@@ -1538,9 +1542,8 @@ def finalizer():
     )
 
     # Creates subscription for elastic-search operator
-    logging_version = config.ENV_DATA['logging_version']
     subscription_yaml = templating.load_yaml(constants.EO_SUB_YAML)
-    subscription_yaml['spec']['channel'] = logging_version
+    subscription_yaml['spec']['channel'] = ocp_version
```

Comment:
what about unGAed ocp 4.4 ? Is there already channel for 4.4? Or will this break deployment over OCP 4.4?

I would go with other approach. Would like to still have possibility to get version from config and if it's not set then use this version dynamically.

Reply:
Anyway if you will have to set this for OCP 4.4 for now to 4.3 before GA if there is no 4.4 channel yet the better place where to specify will be this config file: conf/ocp_version/ocp-4.4-config.yaml